lms/services/canvas.py

```diff
@@ -63,6 +63,38 @@ def public_url_for_file(
 
             return url
 
+    @staticmethod
```

Comment:
I moved this all around the place now and while I don't think static methods in the service is the absolute perfect solution I think is better than some of the alternatives:

- Common functions somewhere in `resources/` and new resource for the sync api. 

This will work but we'll need a new class + methods for every new  caller. I think this approach will add to much friction to reuse.

- Helper module `helpers/canvas.py` 

Helper functions/modules could work. LTIResource is very much a helper class already. But I think adding the helper concept now in the code base has the problem that it's not clear why some canvas functions might be in the helper module, some in LTIResource and some in any of the canvas related services.



As `staticmethods` in the service is not a surprising location. Having them as `staticmethods` allows to use them directly in the service or as standalone functions when you don't want to pull the whole service.